src/test/scala/com/zeta/playthegame/integration/ServerApp.scala

```diff
@@ -0,0 +1,28 @@
+package com.zeta.playthegame.integration
+
+import cats.effect.{ContextShift, Fiber, IO, Timer}
+import org.http4s.{HttpRoutes, Uri}
+import org.http4s.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.scalatest.{BeforeAndAfterAll, TestSuite}
+
+trait ServerApp extends BeforeAndAfterAll { self: TestSuite =>
+
+  implicit val ec                   = scala.concurrent.ExecutionContext.Implicits.global
+  implicit val cs: ContextShift[IO] = IO.contextShift(ec)
+  implicit val timer: Timer[IO]     = IO.timer(ec)
+
+  val baseUri = Uri.unsafeFromString("http://localhost:8080")
+  val router: HttpRoutes[IO]
```

Comment:
La idea es cada IT extienda este trait y define el `servicio` o `routes` que quiera probar, y asi evitar levantar un Server con todas las rutas

Reply:
Bueni pero agregame pure config.